src/internal/raw-interfaces.ts

```diff
@@ -31,12 +31,12 @@ export interface RawSpamInfo {
   classifications: NftSpamClassification[];
 }
 
-// Information on the time at which an NFT was last acquired.
+/** Information on the time at which an NFT was last acquired. */
 export interface RawAcquiredAt {
-  // Timestamp of the block at which an NFT was last acquired.
+  /** Timestamp of the block at which an NFT was last acquired. */
   blockTimestamp: string;
```

Comment:
Missed this the first review but these fields can be undefined in v2 right?

Reply:
yup